src/app/incident/incident-list/incident-list.component.spec.ts

```
@@ -19,6 +20,11 @@ describe('IncidentListComponent', () => {
     }).compileComponents();
   });
 
+  beforeEach(() => {
+    fixture = TestBed.createComponent(IncidentListComponent);
+    component = fixture.componentInstance;
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)